packages/wonder-blocks-dropdown/components/multi-select.test.js

```diff
@@ -56,10 +56,10 @@ describe("MultiSelect", () => {
         opener.simulate("keyup", {keyCode: keyCodes.space});
         expect(select.state("open")).toEqual(false);
 
-        // Shouldn't open with enter
+        // Should open with enter
         opener.simulate("keydown", {keyCode: keyCodes.enter});
         opener.simulate("keyup", {keyCode: keyCodes.enter});
-        expect(select.state("open")).toEqual(false);
+        expect(select.state("open")).toEqual(true);
```

Comment:
This will be an exciting test to have in vite too.

Reply:
Are you thinking screenshot tests for this?

Reply:
Yeah, I think that would be great in the long term - after all, seeing that the dropdown visually appears when we expect it too and looks right is a good thing.